src/config/ConfigManager.cpp

```diff
@@ -525,7 +525,7 @@ CConfigManager::CConfigManager() {
     m_pConfig->addConfigValue("cursor:no_hardware_cursors", Hyprlang::INT{0});
     m_pConfig->addConfigValue("cursor:no_break_fs_vrr", Hyprlang::INT{0});
     m_pConfig->addConfigValue("cursor:min_refresh_rate", Hyprlang::INT{24});
-    m_pConfig->addConfigValue("cursor:hotspot_padding", Hyprlang::INT{1});
+    m_pConfig->addConfigValue("cursor:hotspot_padding", Hyprlang::INT{0});
```

Comment:
good default because it will prevent firefox to not show toolbars when in fullscreen, which statistically is more probable and if users don't research on this matter they won't even know what is wrong. the minimum hotspot should be 0 so that firefox will work properly or else hyprland is unusable.